submissions/MarharytaBoichenko/html_css_popup/sass/main.scss

```diff
@@ -0,0 +1,304 @@
+$primaryColor: #000;
+$mainFont: 'Merriweather', serif;
+$btnColor: #416aff;
+$bcgColor: #ffffff;
+
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+body {
+  min-height: 100%;
+  font-family: $mainFont;
+  font-size: 16px;
+  line-height: 1.71;
+  letter-spacing: 0.03em;
+  color: $primaryColor;
+  background-image: url('../popup-icons/ua.jpeg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+h1,
+h2,
+h3,
+p {
```

Comment:
it is a bad practice to style by HTML tag
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

Reply:
обнуление стилей is reseting, another approach for reseting default browser styles. Yeah, for normalizing it is fine, but don't forget that these styles applies to all the tags, so use it carefully in real projects.  IMHO, you can move it to separate file, name it normalize.css and import here, it would be great, but it is "better to have", not mandatory.